cluster-provision/images/vm-image-builder/create-containerdisk.sh

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env bash
 set -exuo pipefail
 
+# dnf -y install bridge-utils libvirt virt-install qemu-kvm cloud-utils guestfs-tools
```

Comment:
I find it really convenient to have here, in 1 command instead of bullet points, especially when having to run the script on multiple throw-away installs, but I understand it effectively duplicates the docs. Will remove.